backend/app/database/alembic/versions/2025_11_25_1015-b3f7df8cf884_add_dataset_curated_queries.py

```diff
@@ -0,0 +1,68 @@
+"""Create dataset curated queries
+
+Revision ID: b3f7df8cf884
+Revises: 0f590c0c9b20
+Create Date: 2025-11-25 10:15:00.000000
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+from app.shared.model import utcnow
+
+# revision identifiers, used by Alembic.
+revision: str = "b3f7df8cf884"
+down_revision: Union[str, None] = "58c7d09c5798"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "dataset_curated_queries",
+        sa.Column(
```

Comment:
Do we need this? We could also make a combined primary key of:

`output_port_id, sort_order`?